activerecord/lib/active_record/schema_dumper.rb

```diff
@@ -237,6 +250,27 @@ def check_constraints_in_create(table, stream)
         end
       end
 
+      def exclusion_constraints_in_create(table, stream)
+        if (exclusion_constraints = @connection.exclusion_constraints(table)).any?
```

Comment:
do you need to check `@connection.supports_exclusion_constraints?` here?

Reply:
Nope, we shouldn't need to, as [the caller](https://github.com/rails/rails/pull/40224/files#diff-22750093c2231cef8c58e7e94a0a9266267d33bd990c607475e55fe285fccb8aR175) of this private method does that for us!